Libraries/Experimental/SwipeableRow/SwipeableRow.js

```diff
@@ -57,83 +53,72 @@ const RIGHT_SWIPE_BOUNCE_BACK_DURATION = 300;
  */
 const RIGHT_SWIPE_THRESHOLD = 30 * SLOW_SPEED_SWIPE_FACTOR;
 
-type Props = $ReadOnly<{|
+export type SwipableRowProps = $ReadOnly<{|
   children?: ?React.Node,
   isOpen?: ?boolean,
-  maxSwipeDistance?: ?number,
+  maxSwipeDistance: number,
```

Comment:
Why this change?

Reply:
After I converted it to a React class, it being nullable had a lot of type errors like this:

```
undefined [1] is not a number.

 [1]  59│   maxSwipeDistance?: ?number,
        :
     306│     const maxSwipeDistance = IS_RTL
     307│       ? -this.props.maxSwipeDistance
     308│       : this.props.maxSwipeDistance;
     309│     this._animateTo(-maxSwipeDistance, duration);
     310│   }
     311│
     312│   _animateToClosedPosition(duration: number = SWIPE_DURATION): void {
```

But it has a default prop for `maxSwipeDistance`, so I removed the nullability because [the flow docs](https://flow.org/en/docs/react/components/) mention this:

> Note: You don’t need to make foo nullable in your Props type. Flow will make sure that foo is optional if you have a default prop for foo.

When it's a `creatReactClass` again, I should revert this because that notice will no longer be relevant.